discord/interactions.py

```diff
@@ -233,6 +234,11 @@ def permissions(self) -> Permissions:
         """
         return Permissions(self._permissions)
 
+    @property
```

Comment:
This could be a cached property
```suggestion
    @utils.cached_slot_property("_cs_app_permissions")
```